floris/tools/layout_functions.py

```diff
@@ -1,376 +1,323 @@
 
 import math
-
+from typing import (
+    Any,
+    Dict,
+    List,
+    Tuple,
+)
+
+import matplotlib.lines
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 from scipy.spatial.distance import pdist, squareform
 
-
-# Defines a bunch of tools for plotting and manipulating
-# layouts for quick visualizations
+from floris.tools import FlorisInterface
+from floris.utilities import rotate_coordinates_rel_west, wind_delta
 
 
-def visualize_layout(
-    fi,
-    ax=None,
-    show_wake_lines=False,
-    limit_dist_m=None,
-    lim_lines_per_turbine=None,
-    turbine_face_north=False,
-    one_index_turbine=False,
-    black_and_white=False,
-    plot_rotor=False,
-    turbine_names=None
-):
+def plot_turbine_points(
+    fi: FlorisInterface,
+    ax: plt.Axes = None,
+    turbine_indices: List[int] = None,
+    plotting_dict: Dict[str, Any] = {},
+) -> plt.Axes:
     """
-    Make a plot which shows the turbine locations, and important wakes.
+    Plots turbine layout from a FlorisInterface object.
```

Comment:
Might be good to note that this assumed wind direction is 270

Reply:
This doesn't plot any flow so I don't think that is assumed?

Reply:
Note this line is now at https://github.com/NREL/floris/pull/805/files#diff-a12fc22ccce42f15bb7383a32268cb73c30c13ed05556260633ed06fed0c5631R20.
Good point @paulf81 that this doesn't specifically depend on the wind direction. It was initially confusing for me because we currently have a form of the flow visualizations that rotate the turbines and keep the wind coming from the left instead of keeping the turbines fixed and oriented with the cardinal directions (left: west, up: north, etc). So, considering how this function relates to that one, it might be worth mentioning that this one plots the farm layout with the cardinal directions oriented with left/right/up/down. It's redundant, but this nuance got me so it'll probably get someone else.

Reply:
Is this right though?  This plot from example 23, the wind is 300 degrees, and flow and layout visualizations line up:

![image](https://github.com/NREL/floris/assets/6117702/b2e2961f-4f16-4461-b45d-764e79d6257e)

Reply:
Ah you're right. I was thinking about #578, but I found https://github.com/NREL/floris/pull/578#issuecomment-1534131964 that notes the plotting was changed to always keep the layout fixed and instead rotate the wind direction. Disregard this review comment, and thanks for helping me get there!

Reply:
Pleasure!